### DISCUSSION TITLE
Request:下位層テンプレートをナビゲーションウィンドウからのページ作成でも適用したい

### DISCUSSION THREAD

**Discussion by takamasa-s** [growilabs/growi#7704] [Ideas / Requests] [answered]
## Informations
下位層テンプレートの作成後、ページ右上の"作成"ボタンより新規ページを作成することで
テンプレートが反映されたページを作成することができますが、新規ページを左側の
ナビゲーションウィンドウ各章の"＋"ボタンから作成することも場面もよくあります。

**Describe the solution you'd like**
ナビゲーションウィンドウ各章の"＋"ボタンから作成する新規ページにおいても
下位層テンプレートが反映されたページを作成できるようにしていただけると大変助かります。

**Comment by yuki-takei** [comment]
@takamasa-s ご要望ありがとうございます。実装検討します。

**Comment by yuki-takei** [comment, marked as answer]
@takamasa-s [v6.0.15](https://github.com/weseek/growi/releases/tag/v6.0.15) で修正されました。

**Reply by takamasa-s** [reply]
@yuki-takei ご対応いただき大変助かります。ありがとうございました。
